core/src/test/java/org/apache/druid/java/util/common/StringUtilsTest.java

```diff
@@ -160,4 +165,24 @@ public void testURLEncodeSpace()
     Assert.assertEquals(s2, "fff%2Bggg");
     Assert.assertEquals("fff+ggg", StringUtils.urlDecode(s2));
   }
+
+  @Test
+  public void testRepeat()
+  {
+    {
+      Assert.assertEquals("", StringUtils.repeat("foo", 0));
+      Assert.assertEquals("foo", StringUtils.repeat("foo", 1));
+      Assert.assertEquals("foofoofoo", StringUtils.repeat("foo", 3));
+    }
```

Comment:
nit: maybe replace `}` with a blank line? There is no such code style in druid. The same for the `{` and `}` below.

Reply:
Fixed.